extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -372,6 +372,42 @@ tags: {  # <amp-story-cta-layer>
   mandatory_ancestor: "AMP-STORY-PAGE"
   descendant_tag_list: "amp-story-cta-layer-allowed-descendants"
   mandatory_last_child: true
+  reference_points: {
+    tag_spec_name: "AMP-STORY-CTA-LAYER animate-in"
+  }
+}
+tags: {
+  html_format: AMP
+  tag_name: "$REFERENCE_POINT"
+  spec_name: "AMP-STORY-CTA-LAYER animate-in"
+  attrs: {
+    name: "animate-in"
+    value_regex: "drop|"
```

Comment:
We've introduced a repeated `value` field, so this can now be expressed as:

```
  value: "drop"
  value: "fade-in"
  value: "fly-in-bottom"
  ...
```